var/spack/repos/builtin/packages/opencascade/package.py

```diff
@@ -67,59 +74,134 @@ class Opencascade(CMakePackage):
         when="@:7.7.0",
     )
 
+    # Modules, per DAG at https://dev.opencascade.org/doc/refman/html/
+    variant("foundation_classes", default=True, description="Build Foundation Classes module")
```

Comment:
@wdconinc is there a use case where someone might not want to build the foundation classes?

Reply:
Yeah, there isn't. I'll remove that variant.